frontend/src/components/CurrentSongView.tsx

```diff
@@ -29,7 +39,7 @@ function CurrentSongView() {
     !asyncSong.loading &&
     !asyncSong.error &&
     asyncSong.result?.data?.content
-      .replaceAll("[tab]", "")
+      .replaceAll("[*tab]", "")
```

Comment:
what's this for, did something change with the way these are coming over the wire?

Reply:
This was a mistake, I thought we might be able get away with a single `replaceAll` by using a wildcard, it didn't work and I missed it in my review.